storages/storages.go

```diff
@@ -44,8 +44,9 @@ type Storages struct {
 // Backend represents any storage in akubra cluster
 type Backend struct {
 	http.RoundTripper
-	Endpoint url.URL
-	Name     string
+	Endpoint    url.URL
+	Name        string
+	Maintenance bool
```

Comment:
Documentation :exclamation: Add information about this feature in README and godoc here